training.py

```diff
@@ -0,0 +1,65 @@
+from data_preprocessing import load_data, create_sliding_window
+from data_postprocessing import filter_data
+from model_handler import train_xgboost_regressor_model
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.model_selection import train_test_split
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from models import Job
```

Comment:
models have been moved to sprout/models.py

Reply:
Updated then.